go2_robot_sdk/go2_robot_sdk/__init__.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import os
+import logging
+from ament_index_python import get_package_share_directory
+
+logger = logging.getLogger(__name__)
+
+libs_path =  os.path.join(
+        get_package_share_directory('go2_robot_sdk'),
+        'external_lib'
+    )
+sys.path.insert(0, os.path.join(libs_path, 'aioice'))
```

Comment:
maybe i can check if `__init__.py` exists in that folder and throw a fatal log telling to update the submodule.